katdal/visdatav4.py

```diff
@@ -181,8 +181,12 @@ def _before(date):
         self.version = '4.0'
         self.dump_period = attrs['int_time']
         # The CBF dump period is not in the lite RDB version
-        self.cbf_dump_period = attrs.get(
-            'i0_baseline_correlation_products_int_time', None)
+        try:
+            src_stream = attrs['src_streams'][0]
```

Comment:
Is the `wide_b_c_p` always stream 0? And what about narrowband instruments?

Reply:
L0 streams always have exactly one element in `src_streams` - there is no difference between wide and narrow.